From 73ce24d009f42275ab368cec0342d0e303381451 Mon Sep 17 00:00:00 2001
From: Alex Deucher <alexander.deucher@amd.com>
Date: Thu, 13 Apr 2017 18:34:54 -0400
Subject: [PATCH 1194/2094] Revert "drm/dp-helper: DP_TEST_MISC1 should be
 DP_TEST_MISC0"

This was missing the DC change and broke the build.

This reverts commit 3602ea59620848794f4283f317c79f6fe59b80f7.
---
 include/drm/drm_dp_helper.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/drm/drm_dp_helper.h b/include/drm/drm_dp_helper.h
index d6a5015..4b14a76 100644
--- a/include/drm/drm_dp_helper.h
+++ b/include/drm/drm_dp_helper.h
@@ -419,7 +419,7 @@
 
 #define DP_TEST_PATTERN			    0x221
 
-#define DP_TEST_MISC0                       0x232
+#define DP_TEST_MISC1                       0x232
 
 #define DP_TEST_CRC_R_CR		    0x240
 #define DP_TEST_CRC_G_Y			    0x242
-- 
2.7.4

